official-antistasi-community/A3-Antistasi
# Militia offroads & civilian CH-49 Retextures

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Adds retextured FIA offroads to the vanilla factions - color choice depending on environment or what would be faction-suitable.
Adds the Ch-49 in its civilian livery as a purchasable helicopter, for 6500€.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Militia offroads: Load template, observe militia roadblocks
Civilian helicopter: buy the helicopter

********************************************************
Notes:
